documentation/book/proc-deploying-cluster-operator-to-watch-multiple-namespaces.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-to-watch-multiple-namespaces{context}']
+= Deploying the Cluster Operator to watch multiple namespaces
+
+.Prerequisites
+
+* Edit the installation files according to the {PlatformNameNamespace} the Cluster Operator is going to be installed in.
++
+On Linux, use:
++
+[source, subs="+quotes"]
+----
+sed -i 's/namespace: .\*/namespace: _my-namespace_/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+On MacOS, use:
++
+[source, subs="+quotes"]
+----
+sed -i '' 's/namespace: .\*/namespace: _my-namespace_/' install/cluster-operator/*RoleBinding*.yaml
+----
+
+.Procedure
+
+. Edit the file `install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml` and in the environment variable `STRIMZI_NAMESPACE` list all the {PlatformNameNamespaces} where Cluster Operator should watch for resources.
+For example:
++
+[source,yaml,subs="attributes"]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  template:
+    spec:
+      serviceAccountName: strimzi-cluster-operator
+      containers:
+      - name: strimzi-cluster-operator
+        image: {DockerClusterOperator}
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: STRIMZI_NAMESPACE
+          value: myproject,myproject2,myproject3
```

Comment:
You use `myproject,myproject2,myproject3` and few lines below you both to deal with `my-namespace` versus `my-project`. I would suggest to uses use single name everywhere. But if you want to replace it . you should do it here as well.

Reply:
I'll rework all the namespaces in this procedure since it's less than clear. `co-namespace`, `watched-namespace-1` etc. will make it much more obvious imho.